src/test/java/org/fundacionjala/coding/MarcoM_spin_word/TestSpinWord.java

```diff
@@ -0,0 +1,28 @@
+package org.fundacionjala.coding.MarcoM_spin_word;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by Administrator on 5/29/2017.
+ */
+public class TestSpinWord {
+    /**
+     * verify.
+     */
+    @Test
+    public void returnWord() {
+        SpinWorld k = new SpinWorld();
+        assertEquals("abd j", k.reverse("abd j"));
+    }
+
+    /**
+     *verify.
+     */
+    @Test
+    public void returnWordTwo() {
+        SpinWorld k = new SpinWorld();
```

Comment:
You are repeating this code in both tests
Please put them in @before method or convert it to a static method.